packages/design/jobberStyle.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require("fs");
+// eslint-disable-next-line import/no-internal-modules
+const customPropertiesObject = require("./foundation.js");
+
+const customProperties = customPropertiesObject.customProperties;
+const allKeys = Object.keys(customProperties);
+const resolvedCssVars = allKeys.reduce((acc, key) => {
+  const newKey = key.replace("--", "");
+  acc[newKey] = jobberStyle(key);
+  return acc;
+}, {});
+
+const jsonContent =
+  "export const JobberStyle = " + JSON.stringify(resolvedCssVars, undefined, 2);
+
+fs.writeFile("./foundation.js", jsonContent, "utf8", function(err) {
```

Comment:
It looks here like we are editing `foundation.js` in place. I wonder if it would be nicer to have a raw file that is generated from postcssCustomProperties in a temp location then using that to generate a fresh `foundation.js` that way we don't have two scripts generating the same file.

Reply:
We could but the raw file wouldn't be needed for anything else.
Would it be better to have a raw file and at the end of the processing have something to delete the raw file?